src/fare-contracts/FareContractOrReservation.tsx

```diff
@@ -44,8 +46,15 @@ export function FareContractOrReservation({
           sentToOthers={mode === 'sent'}
           onPressDetails={onPressFareContract}
           testID={'ticket' + index}
+          style={styles.fareContract}
         />
       </ErrorBoundary>
     );
   }
 }
+
+const useStyles = StyleSheet.createThemeHook((theme) => ({
+  fareContract: {
+    paddingBottom: theme.spacings.large,
```

Comment:
Do we want this padding to apply to all fare contracts? Could we wrap the SimpleFareContract in a view, and apply padding in the warning screen instead?

Reply:
It's due to removing `withBottomPadding` from FareContractView: https://github.com/AtB-AS/mittatb-app/pull/4391/files#diff-bb41825cddfbc0a8a96c04b2818fc2ce9a9a49d9afe0ad6fade19b1ba9c87aabL104
and we don't want this padding to show for all `SimpleFareContracts`

Reply:
Since removing the overlay, I did as you suggested 😄